input/pagecontent/index.md

```diff
@@ -1,3 +1,8 @@
+<div markdown="1" class="note-to-balloters">
```

Comment:
I think this page should include a reference to CodexRT that builds on the mCODE RT support, and involves some of the same folks credited with the RT support in mCODE.